README.md

```diff
@@ -145,17 +164,15 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
-
+main.cpp
```

Comment:
Thank you! You gave me a nice program (which is saying something as it's written in C++) with a proper state table. But you also handled all the edge cases: running out of words in the middle of the FSM, and testing both positive and negative cases automatically.

# +3 bonus